[2025-05-17T09:31:33.102+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: uhi_index_analytics_pipeline_decompose.process_street manual__2025-05-17T05:12:03.413965+00:00 [queued]>
[2025-05-17T09:31:33.225+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: uhi_index_analytics_pipeline_decompose.process_street manual__2025-05-17T05:12:03.413965+00:00 [queued]>
[2025-05-17T09:31:33.227+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-05-17T09:31:33.232+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-05-17T09:31:33.233+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-05-17T09:31:33.569+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): process_street> on 2025-05-17 05:12:03.413965+00:00
[2025-05-17T09:31:33.739+0000] {standard_task_runner.py:55} INFO - Started process 1173 to run task
[2025-05-17T09:31:33.842+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'uhi_index_analytics_pipeline_decompose', 'process_street', 'manual__2025-05-17T05:12:03.413965+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/uhi_pipeline_dag_decompose.py', '--cfg-path', '/tmp/tmp8tqufnmw']
[2025-05-17T09:31:33.853+0000] {standard_task_runner.py:83} INFO - Job 214: Subtask process_street
[2025-05-17T09:31:35.096+0000] {task_command.py:388} INFO - Running <TaskInstance: uhi_index_analytics_pipeline_decompose.process_street manual__2025-05-17T05:12:03.413965+00:00 [running]> on host ee2bb044a094
[2025-05-17T09:31:35.064+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uhi_index_analytics_pipeline_decompose
AIRFLOW_CTX_TASK_ID=process_street
AIRFLOW_CTX_EXECUTION_DATE=2025-05-17T05:12:03.413965+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-17T05:12:03.413965+00:00
[2025-05-17T09:31:33.997+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/uhi_pipeline_dag_decompose.py", line 85, in process_street_data
    preprocess.input_data_handle(data["street"], type="street")
TypeError: 'NoneType' object is not subscriptable
[2025-05-17T09:31:34.093+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=uhi_index_analytics_pipeline_decompose, task_id=process_street, execution_date=20250517T051203, start_date=20250517T093133, end_date=20250517T093134
[2025-05-17T09:31:34.181+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 214 for task process_street ('NoneType' object is not subscriptable; 1173)
[2025-05-17T09:31:34.325+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-05-17T09:31:34.823+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
